packages/decompress/decompress.0.1/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+maintainer: "dinosaure <romain.calascibetta@gmail.com>"
+authors: "dinosaure <romain.calascibetta@gmail.com>"
+homepage: "https://github.com/oklm-wsh/Decompress"
+bug-reports: "https://github.com/oklm-wsh/Decompress/issues"
+dev-repo: "https://github.com/oklm-wsh/Decompress"
```

Comment:
Could you please change the `dev-repo` URL to "https://github.com/oklm-wsh/Decompress.git"?